test/integration/api/submissions.js

```diff
@@ -267,13 +267,13 @@ describe('api: /forms/:id/submissions', () => {
             zipStreamToFiles(asAlice.get('/v1/projects/1/forms/simple/submissions.csv.zip'), (result) => {
               result.filenames.should.eql([ 'simple.csv' ]);
               const csv = result['simple.csv'].split('\n').map((row) => row.split(','));
-              csv[0].should.eql([ 'SubmissionDate', 'meta-instanceID', 'name', 'age', 'KEY' ]);
+              csv[0].should.eql([ 'SubmissionDate', 'meta-instanceID', 'name', 'age', 'KEY', 'SubmitterID', 'SubmitterName' ]);
               csv[1].shift().should.be.an.recentIsoDate();
-              csv[1].should.eql([ 'three','Chelsea','38','three' ]);
+              csv[1].should.eql([ 'three','Chelsea','38','three', '5', 'Alice' ]);
```

Comment:
Am I right in thinking that the CSV includes submitter ID and name only if extended metadata is requested? Or does the CSV include submitter ID and name all the time? This request doesn't seem to request extended metadata, so I was surprised that the resulting CSV includes submitter ID and name.

Reply:
Also, if extended metadata is used in this way, it might be useful to have one integration test that requests extended metadata and one that doesn't.

Reply:
i think it’s relatively impractical to gate the presence of columns on a header field. how would you ask the browser to download such an object?

Reply:
Ah I see, I was understanding the addition of `QueryOptions.extended` in the code as meaning that extended metadata was being used to gate. I see now that the CSV endpoint just always passes `QueryOptions.extended` in order to get the data it needs to always add submitter ID and name to the CSV. I agree that not gating is preferable!